job-service-acceptance-tests/src/test/java/com/hpe/caf/jobservice/acceptance/JobServiceDatabaseUtil.java

```diff
@@ -95,8 +96,8 @@ public static void assertJobRowExists(final String jobId) throws SQLException
             st.setString(1, jobId);
             final ResultSet jobRS = st.executeQuery();
             jobRS.next();
-            Assert.assertEquals(jobRS.getString(1), jobId);
-            Assert.assertTrue(!jobRS.next());
+            Assert.assertEquals(jobRS.getString(1), jobId, "Job "+jobId+" exists in job table");
+            Assert.assertTrue(!jobRS.next(), "No dupliacte job "+jobId+" exists in job table");
```

Comment:
Typo here
```suggestion
            Assert.assertTrue(!jobRS.next(), "No duplicate job "+jobId+" exists in job table");
```

Reply:
I have updated the assertion messages so that the messages could be more significant when it fails.